src/index.js

```diff
@@ -0,0 +1,79 @@
+import 'lodash';
+import './style.css';
+import elementChanges from './check.js';
+import icon from './delete.png';
+import removeAll from './remove.js';
+import Add from './add.js';
+
```

Comment:
- [ ] We are trying to write modularized code. According to the project requirements (see image below), there should be different defined functions in different files for the following actions:
  - adding a new task (add a new element to the array)
  - editing task descriptions
  - deleting a task
  - "Clear all completed" button.

Write now you only have functions for adding and removing completed tasks. Kindly refactor your `index.js` file and extract functions for **deleting individual tasks** and **editing tasks** into respective files.

![meq-req](https://user-images.githubusercontent.com/36542986/144603578-c711f3d7-34de-48fb-a215-bb5a6b957ba7.jpg)